src/forms.css

```diff
@@ -444,14 +444,14 @@ because it seems to be more cross-browser consistent with all HTML5 input types
   height: 20px;
   border-radius: 50%;
   background: var(--white);
-  border: 2px solid var(--default-interactive-color);
+  border: 2px solid var(--default-secondary-interactive-color);
   cursor: pointer;
 }
 
 /* range thumb:hover */
-.range:hover::-webkit-slider-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-ms-thumb { border-color: var(--default-interactive-color-dark); }
-.range:hover::-moz-range-thumb { border-color: var(--default-interactive-color-dark); }
+.range:hover::-webkit-slider-thumb { border-color: var(--gray); }
+.range:hover::-ms-thumb { border-color: var(--gray); }
+.range:hover::-moz-range-thumb { border-color: var(--gray); }
```

Comment:
So I was thinking:

- for textboxes, blue on hover to  show more contrast 
- checkbox, radio button, and switches, blue on hover to show more contrast between different select states.
- for select and range input, dark gray on hover, blue seems to be too much